src/libaktualizr/package_manager/dockerapp_bundles.cc

```diff
@@ -130,6 +130,11 @@ data::InstallationResult DockerAppBundles::install(const Uptane::Target &target)
   data::InstallationResult res;
   Uptane::Target current = OstreeManager::getCurrent();
   if (current.sha256Hash() != target.sha256Hash()) {
+    // notify the bootloader before installation happens as it is not atomic
+    // and a false notification doesn't hurt with rollback support in place
+    if (bootloader_ != nullptr) {
+      bootloader_->updateNotify();
+    }
```

Comment:
Looks like we can now replace this call with `OstreeManager::updateNotify()`, which is now basically `void updateNotify() override { bootloader_->updateNotify(); };`

Reply:
ugh -just accidentally pushed this here instead of into my review branch. its been fixed :/